doc/common/chapter00.md

```diff
@@ -53,7 +41,7 @@ One of major challenges holding back the more rapid and widespread adoption of V
 To put this effort into perspective, over the past few years, the telecom industry has been going through a massive technology revolution by embracing software defined networking and cloud architecture principles, in pursuit of the goal of achieving more flexibility, agility and operational efficiency. At a high level, the main objective of NFV (Network Function Virtualisation) is the ability to use general purpose standard COTS (Commercial off the Shelf) compute, memory and storage hardware platforms to run multiple Virtualised Network Functions.  Earlier common infrastructure models built on the previous assumption that networking applications are typically built on discrete hardware, do not offer the level of flexibility and agility needed for the support of newer networking technologies such as 5G, intelligent networks and Edge computing.  By running network applications as software rather than on purpose-built hardware, as it has been done since the early 1990’s, the operators aspire to realize operational efficiencies, and capital expense savings.  These Software Defined Network (SDN) applications are increasingly being used by telecom operators to support their internal and customer facing network infrastructures.  The need for a common model across the industry to facilitate more rapid adoption is clear.
```

Comment:
Is there a reference to "Earlier common infrastructure models ..." also what is meant by "discrete hardware" (the next sentence refers to "custom hardware"). 

"By running network applications as software rather than on purpose-built hardware .." since the word "rather" is in the sentence, then something is missing like in ".. software <on xyz> rather than ..". BTW, I think SW on custom hardware to implement network functions (like switching) started in the 1970's.

Reply:
it might be better to create a different PR to address this since this is not the scope of this PR.